{
    "type": "bundle",
    "id": "bundle--48487d44-aeef-4c7b-82ac-e8314851f560",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--d39f8260-8e88-409d-ac21-e01eb3f673e5",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2025-06-01T00:30:08.813106Z",
            "modified": "2025-06-01T00:30:08.813106Z",
            "name": "CVE-2018-25111",
            "description": "django-helpdesk before 1.0.0 allows Sensitive Data Exposure because of os.umask(0) in models.py.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2018-25111"
                }
            ]
        }
    ]
}